### ISSUE TITLE
add topics

### ISSUE THREAD

**Issue by Beliavsky** [Kris-SF/public_projects#4]
I suggest adding topics such as `options`, `stock-options`, `finance`, `quantitative-finance`, `delta-hedging`, `option-pricing`, `hedging` in the About section at https://github.com/Kris-SF/public_projects

**Comment by Kris-SF** [comment]
Curious why? For searchability?

**Comment by Beliavsky** [comment]
Yes, so your project will appear under https://github.com/search?q=topic%3Aoption-pricing&type=repositories for example. It's also a quick way to describe what your project is about. How to add topics is described at https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/classifying-your-repository-with-topics .
